.github/workflows/bundle_size.yml

```
@@ -1,26 +1,7 @@
-name: Compressed Size
+name: Bundle size
```

```suggestion
name: Nuxt bundle size
```

Or "frontend", perhaps? We should be as specific as possible in a monorepo context